include/oxen/quic/opt.hpp

```diff
@@ -202,5 +202,83 @@ namespace oxen::quic
                     _hook = nullptr;
             }
         };
+    }  // namespace opt
+
+    using gtls_db_validate_cb = std::function<bool(gtls_ticket_ptr, time_t)>;
+    using gtls_db_get_cb = std::function<gtls_ticket_ptr(ustring_view)>;
+    using gtls_db_put_cb = std::function<void(gtls_ticket_ptr, time_t)>;
+
+    namespace opt
+    {
+        /** 0-RTT ticketing:
+                The application has two choices in managing 0-RTT ticket storage and the server-side anti-replay db. Passing
+            either the default-constructed or expiry window constructed struct will signal to the endpoints that all storage
+            will happen internally.
+                The application can also pass ALL callbacks and take full responsiblity for management. In doing so, the user
+            must still pass an expiry window value, as that is given directly to gnutls. The following details enumerate the
```

Comment:
Can you elaborate here on what the expiry window does, and how I should choose it?